src/main.rs

```diff
@@ -49,7 +50,11 @@ fn calculate_safety_status(
     has_carrot: bool,
     friends_nearby: i32,
 ) -> bool {
-    todo!()
+    if wolves_nearby == false && day_time == true || has_carrot == true || friends_nearby > 3 {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
This is pretty good! But you can actually simplify this quite a lot by using boolean logic all on one line. It might look something like `a || (b && c)`.